src/spikeinterface/core/generate.py

```diff
@@ -177,6 +177,65 @@ def generate_sorting(
     return sorting
 
 
+def generate_injected_sorting(
+    sorting: BaseSorting,
+    num_samples: List[int],
+    max_injected_per_unit: int = 1000,
+    injected_rate: float = 0.05,
+    refractory_period_ms: float = 1.5,
+) -> NumpySorting:
+    """
+    Generates a sorting object with injected spikes from an existing sorting object.
```

Comment:
Can we take this cahnce to improve this docstring a little bit. I don't really understand what this object does.

Is it a sorting object that has spikes sampled from another sorting?